modules/structs/package.go

```diff
@@ -23,8 +23,8 @@ type Package struct {
 
 // PackageFile represents a package file
 type PackageFile struct {
-	ID         int64 `json:"id"`
-	Size       int64
+	ID         int64  `json:"id"`
+	Size       int64  `json:"size"`
```

Comment:
Do we see this as a breaking change?

Reply:
I think yes. The previous result is `{"Size": 0}`, now it's `{"size":0}`.

Reply:
I've just updated the PR description.

Reply:
Is the break change necessary?

Reply:
Not for this PR but should be done anyway.